### DISCUSSION TITLE
Elasticsearch: Lucene syntax highlighting in logs query field in Explore

### DISCUSSION THREAD

**Discussion by marefr** [grafana/grafana#42320] [Observability]
**What would you like to be added**:
Would be nice to have Elasticsearch logs query field in Explore highlighting the lucene syntax.

**Why is this needed**:
Makes it hopefully easier to see either what query you're using or either if the query is not properly formatted.

It was discussed/thought of a bit initially when implementing #17605 so adding this so that we can see if others are interested in this feature.

**Comment by kaydelaney** [comment]
https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-highlighting.html
Worth taking a look at for whoever ends up implementing this.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
